db/schema.sql

```diff
@@ -0,0 +1,80 @@
+CREATE SCHEMA [IF NOT EXISTS] qa_schema;
+
+SET search_path TO qa_schema, public;
+
+CREATE TABLE questions (
+  question_id INTEGER NOT NULL PRIMARY KEY,
+  product_id INTEGER,
```

Comment:
Should product_id be required? What would happen if we tried to insert a record without a product_id?

Reply:
I think this would be the foreign key that's going to be used to pull the questions. Without the product_id, it wouldn't be able to pull the question right?

Reply:
So a foreign key would have to reference another table which would only have one column (product_id) so it doesn't make sense to have that table. I can just query by value. But yes, I think the product_id should ideally be not null. I will update and redo the ETL.

Reply:
Updated! Merging with new commit.